script.js

```diff
@@ -299,3 +310,45 @@ workMobile.forEach((work) => {
   });
   mobileWork.appendChild(item);
 });
+
+// form validatio
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224448698-af63235d-68dd-4c81-b4c1-e172b1428a53.png)

**[Optional]**

- [ ] It would be nice if you remove this section description to make your readme look clean and professional 😊 These section descriptions are just instructions for you when you will make your readme file and after making your readme file you are supposed to remove all of them because these are unnecessary for the viewers of your repository 🤔